.github/workflows/frontend.yml

```diff
@@ -124,7 +117,7 @@ jobs:
         run: yarn tsc -p config/tsconfig.ci.json
 
   frontend-jest-tests:
-    if: needs.files-changed.outputs.frontend == 'true'
+    if: github.ref == 'refs/heads/master' || needs.files-changed.outputs.testable_rules_changed == 'true'
```

Comment:
Is this intentional to always run jest tests on master even if no frontend files were changed?

Reply:
good point, this could only look at the `testable_modified_rules_changed`